packages/app/src/common/constants.ts

```diff
@@ -1,6 +1,9 @@
 import { Subdomains } from '@blockstack/keychain';
+import { StacksTestnet } from '@blockstack/stacks-transactions';
 
 export const gaiaUrl = 'https://hub.blockstack.org';
+export const defaultStacksNetwork = new StacksTestnet();
+defaultStacksNetwork.coreApiUrl = 'https://stacks-node-api.blockstack.org';
```

Comment:
These network classes should take a constructor argument, so we don't have to imperatively modify the properties like this. Will make an issue in the new monorepo for this.